packages/dropdowns/src/elements/Multiselect/Multiselect.tsx

```diff
@@ -14,26 +14,22 @@ import { KEY_CODES, useCombinedRefs } from '@zendeskgarden/container-utilities';
 import { isRtl, withTheme } from '@zendeskgarden/react-theming';
 import {
   StyledMultiselectInput,
-  StyledSelect,
   StyledItemWrapper,
   StyledMoreAnchor,
-  VALIDATION
+  StyledMultiSelect
 } from '../../styled';
+import { VALIDATION } from '../../utils/validation';
 import useDropdownContext from '../../utils/useDropdownContext';
 import useFieldContext from '../../utils/useFieldContext';
 import { REMOVE_ITEM_STATE_TYPE } from '../Dropdown/Dropdown';
 
 interface IMultiselectProps {
-  /** Applies flex layout to support MediaFigure components */
-  mediaLayout?: boolean;
   isCompact?: boolean;
   /** Removes all borders and styling */
   isBare?: boolean;
   disabled?: boolean;
-  isFocused?: boolean;
   /** Applies inset `box-shadow` styling on focus */
   focusInset?: boolean;
-  isHovered?: boolean;
   /** Displays select open state */
   isOpen?: boolean;
   placeholder?: string;
```

Comment:
I would say since the design assets still include it in Sketch, for the interim, I'd vote for making it available across the line until we explicitly outlaw it or provide accessible guidelines for its use.

Reply:
Just to clarify, these components don't offer the placeholder value as the `<input>` element isn't immediately visible before the element is opened.

For the Multiselect the input is shown when there are no items selected.

Reply:
That makes sense – probably fine to leave as-is for now.